### ISSUE TITLE
tf_give_personal_radio_to_regular_soldier  Doesn't work

### ISSUE THREAD

**Issue by john681611** [michail-nikolaev/task-force-arma-3-radio#1160]
**Arma 3 Version:** `1.64` (stable)
**CBA Version:** `3.1.1.161012` (stable)
**Task Force Radio Version:** `0.9.12` (stable)

**Mods:**
- `@CBA_A3`
-'@ACE3'
- `@task_force_radio

**Description:**
tf_give_personal_radio_to_regular_soldier no longer works when using in init.sqf
using; 

`tf_give_personal_radio_to_regular_soldier=true;`

**Where did the issue occur?**
- Dedicated + Self-Hosted Multiplayer

**Comment by dedmen** [comment]
Example Mission pbo please.

**Comment by dedmen** [comment]
Are you sure you are running 0.9.10?

**Comment by john681611** [comment]
no 0.9.12 Sorry I've not given you a example mission I'm deep in persistent mission dev atm

**Comment by dedmen** [comment]
You can't just set that Variable anymore in 0.9.12. I've already written than in BIF.
Either set Serverside settings or use these scripts
https://forums.bistudio.com/topic/159393-task-force-arrowhead-radio/?p=3106767

**Comment by john681611** [comment]
I'm Trying to set the values in init.sqf like so
```
["tf_give_personal_radio_to_regular_soldier", true, true] call CBA_settings_fnc_set;
["tf_no_auto_long_range_radio", true, true] call CBA_settings_fnc_set;
["tf_give_microdagr_to_soldier", false, true] call CBA_settings_fnc_set;
```
Im just getting nothing working and this warning
```
21:53:12 [CBA] (settings) WARNING: Value true is invalid for setting "tf_give_personal_radio_to_regular_soldier". x\cba\addons\settings\fnc_set.sqf:36
21:53:12 [CBA] (settings) WARNING: Value true is invalid for setting "tf_no_auto_long_range_radio". x\cba\addons\settings\fnc_set.sqf:36
21:53:12 [CBA] (settings) WARNING: Value false is invalid for setting "tf_give_microdagr_to_soldier". x\cba\addons\settings\fnc_set.sqf:36
```

**Comment by dedmen** [comment]
You need to call after (time > 2)
just `waitUntil {time > 2;};` before your script
Will be fixed in 1.0.
Or just use The way its supposed to work. Set stuff in the ingame settings menu.
